apis/v1alpha2/policy.go

```diff
@@ -39,9 +39,6 @@ const (
 type PolicyConditionType string
 
 const (
-	// PolicyActive represents the condition of the Policy admission
-	// webhook been registered
-	PolicyActive PolicyConditionType = "PolicyActive"
```

Comment:
Please keep this one around. I forgot to add the condition set to `True` on the refactor. This is the condition our users can wait for when they create a new policy, this condition will be true when we register the webhooks.

I think we can do that on a follow up PR, but I would not remove this condition.

Reply:
Okay